### PR TITLE
rubocop config

### PR DESCRIPTION
**Pull request by szmo**
https://trello.com/c/R9o1h4fs/4-doda%C4%87-config-dla-rubocopa

### CODE REVIEW

File: rails_app/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# The guiding principle of classes is SRP, SRP can't be accurately measured by LoC
+Metrics/ClassLength:
+  Max: 1500
```
Comment (evemorgen) [inline rails_app/.rubocop.yml:3]: Damn boy, thats a lot of lines for a single class

File: rails_app/.rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Rails:
+  Enabled: true
+Rails/Delegate:
+  Enabled: false
+Rails/HttpPositionalArguments:
+  Enabled: false
+
+# Most readable form.
+Style/AlignHash:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml:22]: Could not find these in [docs](https://rubocop.readthedocs.io/en/latest/cops_style/), typo or sth?

File: rails_app/.rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  EnforcedHashRocketStyle: table
+  EnforcedColonStyle: table
+
+Style/AlignParameters:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml]: Same as before, didn't find that one in [docs](https://rubocop.readthedocs.io/en/latest/cops_style/)

File: rails_app/.rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  EnforcedStyle: braces_for_chaining
+
+Style/BracesAroundHashParameters:
+  EnforcedStyle: context_dependent
+
+# Check with yard instead.
+Style/Documentation:
+  Enabled: false
+
+Style/EmptyElse:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml]: Sure about that? I think this looks awful
```
if condition
  statement
else
end
```

File: rails_app/.rubocop.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  Enabled: false
+
+Style/EmptyLinesAroundClassBody:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml]: 404, not found

File: rails_app/.rubocop.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  Enabled: false
+
+Style/IndentArray:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml]: 404

File: rails_app/.rubocop.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  EnforcedStyle: consistent
+
+Style/IndentHash:
```
Comment (evemorgen) [inline rails_app/.rubocop.yml]: 404

**Review by evemorgen — CHANGES_REQUESTED** [review]
Please reply to my previous comments

File: rails_app/.rubocop.yml (hunk 2 above)
Comment (szmo) [reply rails_app/.rubocop.yml:22]: http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/AlignHash

Comment (evemorgen) [reply rails_app/.rubocop.yml:22]: so they suck about keeping their docs huh, sorry for that, deleting rest of comments now

File: rails_app/.rubocop.yml (hunk 1 above)
Comment (szmo) [reply rails_app/.rubocop.yml:3]: thought its self explanatory comment above

**Review by evemorgen — APPROVED** [review]

**Pull request merged**